### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rohitdaryanani**
Issue #1

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+1. [Fork](https://help.github.com/fork-a-repo/) the project, clone your fork,
+   and configure the remotes:
+
+   ```bash
+   # Clone your fork of the repo into the current directory
+   git clone https://github.com/<your-username>/<repo-name>
+   # Navigate to the newly cloned directory
+   cd <repo-name>
+   # Assign the original repo to a remote called "upstream"
+   git remote add upstream https://github.com/mazipan/vue-currency-filter
```
Comment (mazipan) [inline CONTRIBUTING.md]: Typo for `vue-currency-filter`

Comment (mazipan) [reply CONTRIBUTING.md]: It should be `git remote add upstream https://github.com/mazipan/phpid-uxd-landingpage-prototype`

**Comment by mazipan** [comment]
We should add link in `readme.md` too I think @rohitdaryanani

File: CONTRIBUTING.md (hunk 1 above)
Comment (rohitdaryanani) [reply CONTRIBUTING.md]: ow sorry will update pr

**Comment by rohitdaryanani** [comment]
@mazipan sorry what link should we add in the `readme.md`?

**Comment by mazipan** [comment]
Link to `contributing.md`, you can follow https://github.com/mazipan/vue-currency-filter/pull/5/files

**Comment by rohitdaryanani** [comment]
ah okay added CONTRIBUTING.md in README.md let me know if this is correct

**Pull request merged**